plugin/core/open.py

```diff
@@ -15,6 +15,19 @@
 opening_files = {}  # type: Dict[str, Tuple[Promise[Optional[sublime.View]], ResolveFunc[Optional[sublime.View]]]]
 
 
+def _return_existing_view(flags: int, open_file_group: int, active_group: int, specified_group: int) -> bool:
+    if specified_group > -1:
+        return False
+    open_side_by_side = bool(flags & (sublime.ADD_TO_SELECTION | sublime.REPLACE_MRU))
+    if open_side_by_side:
```

Comment:
This could be put directly into the if-condition without initializing a new variable for it.